Breeze/src/Breeze.Wallet/Models/WalletHistoryModel.cs

```diff
@@ -13,15 +14,15 @@ public class WalletHistoryModel
 	public class TransactionItem
 	{
 		[JsonProperty(PropertyName = "txId")]
-		public string TransactionId { get; set; }
+		public uint256 TransactionId { get; set; }
 
 		[JsonProperty(PropertyName = "amount")]
 		public Money Amount { get; set; }
 
 		[JsonProperty(PropertyName = "confirmed")]
-		public Money Confirmed { get; set; }
+		public bool Confirmed { get; set; }
 
 		[JsonProperty(PropertyName = "timestamp")]
-		public string Timestamp { get; set; }
+		public DateTimeOffset Timestamp { get; set; }
 	}
```

Comment:
The timestamp will be formatted in a pre-agreed string format before being send to the front end.